core/model.py

```diff
@@ -56,23 +44,32 @@ def __init__(self, model_file=PATH_TO_CKPT, label_file=PATH_TO_LABELS):
         self.category_index = category_index
         self.categories = categories
 
-    def predict(self, imageRaw, threshold):  #was origninally run_inference_for_single_image
-        image = preprocess_image(imageRaw)
+    def _read_image(self, image_data):
```

Comment:
Maybe this can just go into `_pre_process`?

Reply:
It's called [here](https://github.com/IBM/MAX-Object-Detector/pull/37/files#diff-63d534adc2afb1d599734dfd51ebf7b0R40) in the API